metabrainz/templates/index/funding.json

```diff
@@ -0,0 +1,233 @@
+        "phone": "001-805-459-0815",
+        "description": "The MetaBrainz Foundation is a non-profit organization dedicated to creating and maintaining open data projects that provide reliable, structured metadata for various forms of media. Established in 2004 by Robert Kaye, MetaBrainz seeks to build free and accessible knowledge repositories while fostering collaboration within an open community. The foundation operates under an open data and open-source ethos, which means its data and tools are freely available for public use, distribution, and modification. This openness encourages developers, researchers, and organizations to contribute and create derivative projects, ultimately enriching the ecosystem around open media metadata.\nAmong the primary projects under MetaBrainz is MusicBrainz, an extensive music database offering metadata on artists, albums, tracks, and release information. Created in response to the proprietary nature of early music metadata services, MusicBrainz empowers users to contribute to and curate an accurate, user-driven music knowledge base. With applications ranging from digital music libraries to tagging software like Picard, MusicBrainz has become a foundational resource for music metadata, attracting contributions from users, as well as support from major companies like Google and the BBC.\nAnother notable project, ListenBrainz, focuses on tracking music listening habits. Inspired by projects like Last.fm, ListenBrainz records and analyzes listening data from users who can share this information to explore trends, recommendations, and personal analytics. As with other MetaBrainz projects, ListenBrainz data is open and accessible, allowing researchers and developers to build upon this data for music recommendations, research, and social features.\nBookBrainz, the foundation’s third major project, is an open database for book metadata, following a similar structure to MusicBrainz but tailored for literature. BookBrainz aims to catalog authors, works, editions, and other entities in the literary world. This data is freely accessible, making it an invaluable resource for libraries, researchers, and readers who want a comprehensive, community-driven book database.\nMetaBrainz Foundation’s commitment to open-source software is integral to its mission. All software it develops, including the tools, APIs, and data structures, is released under open-source licenses. This transparency ensures the data remains free for everyone, regardless of commercial or non-commercial use, while encouraging innovation and contributions from a global user base.\nMetaBrainz is primarily funded through partnerships and sponsorships, relying on support from institutions like Internet Archive and donations from its user community. This structure allows MetaBrainz to remain independent and committed to its mission of providing high-quality, freely available data to benefit both users and the digital information landscape.",
```

Comment:
The description currently fails validation as it is too long (classic LLM  :rofl:)
Suggested edit that fits in the 2000 chars limit:
```suggestion
        "description": "The MetaBrainz Foundation is a non-profit organization dedicated to creating and maintaining open data projects that provide reliable, structured metadata for various forms of media. Established in 2004 by Robert Kaye, MetaBrainz seeks to build free and accessible knowledge repositories while fostering collaboration within an open community. The foundation operates under an open data and open-source ethos, with its data and tools freely availaThe MetaBrainz Foundation is a non-profit organization founded in 2004 by Robert Kaye, creates and maintains open data projects that provide reliable structured metadata for various forms of media.It seeks to build free and accessible knowledge repositories and foster collaboration within an open community. The foundation operates under an open data and open-source ethos, with its data and tools freely available for public use, distribution, and modification. We encourage developers, researchers, and organizations to contribute and create derivative projects, ultimately enriching the ecosystem around open media metadata.\nMetaBrainz's flagship project, MusicBrainz, is an extensive, user-driven music database for artist, album, track, and release information. It supports applications like Picard for digital music libraries tagging and serves as a foundational resource with contributions from users and support from companies like Google and the BBC.\nListenBrainz focuses on recording and analyzing music listening habits, inspired by projects like Last.fm. Users can use these insights to explore trends and personalized recommendations.\nBookBrainz is an open database for book metadata, similar to MusicBrainz but tailored for literature, cataloging authors, works, editions, and other entities in the literary world. This open data is invaluable for libraries, researchers, and readers seeking a community-driven book database.\nThe MetaBrainz Foundation’s commitment to open-source is integral to its mission. All software including the tools, APIs, and data structures, is released under open-source licenses. This transparency ensures the data remains free for everyone, regardless of commercial or non-commercial use, while encouraging contributions from a global user base.\nMetaBrainz is funded by partnerships, sponsorships, and community donations, allowing it to remain independent and committed to providing high-quality, free data for users and the digital landscape.",
```

Reply:
The other option is to remove all the projects-specific stuff (maybe reduced to a very short overview) and keep the rest as-is.

Reply:
I fumbled somewhere and copied the wrong one I think.

Here's it:


The MetaBrainz Foundation is a non-profit organization founded in 2004 by Robert Kaye, creates and maintains open data projects that provide reliable structured metadata for various forms of media.It seeks to build free and accessible knowledge repositories and foster collaboration within an open community. The foundation operates under an open data and open-source ethos, with its data and tools freely available for public use, distribution, and modification. We encourage developers, researchers, and organizations to contribute and create derivative projects, ultimately enriching the ecosystem around open media metadata.\nMetaBrainz's flagship project, MusicBrainz, is an extensive, user-driven music database for artist, album, track, and release information. It supports applications like Picard for digital music libraries tagging and serves as a foundational resource with contributions from users and support from companies like Google and the BBC.\nListenBrainz focuses on recording and analyzing music listening habits, inspired by projects like Last.fm. Users can use these insights to explore trends and personalized recommendations.\nBookBrainz is an open database for book metadata, similar to MusicBrainz but tailored for literature, cataloging authors, works, editions, and other entities in the literary world. This open data is invaluable for libraries, researchers, and readers seeking a community-driven book database.\nThe MetaBrainz Foundation’s commitment to open-source is integral to its mission. All software including the tools, APIs, and data structures, is released under open-source licenses. This transparency ensures the data remains free for everyone, regardless of commercial or non-commercial use, while encouraging contributions from a global user base.\nMetaBrainz is funded by partnerships, sponsorships, and community donations, allowing it to remain independent and committed to providing high-quality, free data for users and the digital landscape.